install.sh

```diff
@@ -262,6 +262,14 @@ setup_binary() {
     info "Installing k3s to ${BIN_DIR}/k3s"
     $SUDO chown root:root ${TMP_BIN}
     $SUDO mv -f ${TMP_BIN} ${BIN_DIR}/k3s
+
+    if [ "Disabled" != `getenforce` ]; then
```

Comment:
What happens if selinux/getenforce is not installed?

Reply:
I added a check for getenforce